conanfile.txt

```diff
@@ -1,8 +1,8 @@
 [requires]
-Boost/1.62.0@lasote/stable
-hdf5/1.10.1@eugenwintersberger/testing
-gtest/1.8.0@conan/stable
-zlib/1.2.8@conan/stable
+Boost/1.65.1-dm1@ess-dmsc/stable
+hdf5/1.10.1-dm1@ess-dmsc/stable
+gtest/3121b20-dm1@ess-dmsc/testing
```

Comment:
It looks like this package for gtest does something odd. It adds a `d` suffix to the library names when building in debug mode. This may cause the configuration problem on Debian 8 and 9 when building with Conan.